ga4gh/server/frontend.py

```diff
@@ -978,7 +997,7 @@ def login_oidc():
     """
 
     base_url = _generate_base_url()
-    redirect_from = flask.request.args.get('redirectUri', type=str).replace(base_url, '')
+    redirect_from = flask.request.args.get('redirectUri','',type=str).replace(base_url, '')
```

Comment:
To ensure the consistency of the styling, we might want to add a space after commas.